projects/storefrontstyles/scss/components/product/_star-rating.scss

```diff
@@ -9,6 +13,10 @@
     font-size: 20px;
     margin: 0 2.5px;
 
+    @include media-breakpoint-down(sm) {
+      font-size: 48px;
```

Comment:
@morganm58 Although it passes validation, this seems so big it takes the focal point away from the product. Please validate with designers and @Xymmer 

![Screen Shot 2022-06-21 at 3 49 05 PM](https://user-images.githubusercontent.com/15113219/174886459-afc37a7f-0efe-4ae0-916c-110d98e075c3.png)

Here's an alternative using padding only

![Screen Shot 2022-06-21 at 3 47 31 PM](https://user-images.githubusercontent.com/15113219/174886643-01b8c588-78b5-49cd-a5bb-afab65a48d8e.png)